Dockerfile-Swift4.1

```diff
@@ -1,9 +1,7 @@
 FROM norionomura/swift:swift-4.1-branch
 
-RUN apt-get update && apt-get install -y wget apt-transport-https software-properties-common python-software-properties && \
+RUN apt-get update && apt-get install -y \wget apt-transport-https software-properties-common python-software-properties libmysqlclient20 libmysqlclient-dev cstack openssl && \
```

Comment:
Does this work? I believe that in order to install (at least) cstack, you need to add the Vapor repo which happens on line 5 (and 4)

Reply:
It didn't ;)